contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1589,7 +1651,7 @@ static bool
 is_nullable_index(IndexStmt *stmt)
```

Comment:
Same for this function - changed the definition of the function. Is that expected? Also, could you please add a top-level comment what the function is trying to do?

Reply:
Yes it was expected, as is_nullable_index was initially returning true only if all the columns in the index is null else false. But expected behaviour should be return true if atleast one of the column is null else false(if all the columns are not null)

Reply:
Added a top level comment